src/libraries/System.Composition.TypedParts/src/System.Composition.TypedParts.csproj

```diff
@@ -18,6 +18,8 @@ System.Composition.Hosting.ContainerConfiguration</PackageDescription>
     <Compile Include="System\Composition\Debugging\ContainerConfigurationDebuggerProxy.cs" />
     <Compile Include="System\Composition\Debugging\DiscoveredPartDebuggerProxy.cs" />
     <Compile Include="System\Composition\Hosting\ContainerConfiguration.cs" />
```

Comment:
@ericstj  this is a change in OOB library. The change is for .NET 7.0. Do we need to add anything to this csproj to ensure this library will release update?

Reply:
Nothing needs to be done to ship this in `main`, we ship all libraries in the latest release.  Only during servicing do we need to enable components for shipping.